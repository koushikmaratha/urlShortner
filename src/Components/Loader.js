import React from 'react';

const Loader = () => {
    return <span style={{ fontSize: '3em' }}><i className="fas fa-spinner fa-spin"></i></span>;
};

export default Loader;