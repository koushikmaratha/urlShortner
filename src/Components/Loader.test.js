import React from 'react';
import { shallow } from 'enzyme';
import Loader from './Loader';

describe('render Loader Component',()=>{
    let wrapper;
    beforeEach(()=>(shallow(<Loader />)));

    it('Tests the SnapShot',()=>{
        expect(wrapper).toMatchSnapshot();
    })
})