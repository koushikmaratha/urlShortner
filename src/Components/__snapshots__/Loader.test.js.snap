// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Loader Component Tests the SnapShot 1`] = `undefined`;
